Sources/Turf/Info.plist

```diff
@@ -15,9 +15,9 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.1.0</string>
+	<string>1.1.1</string>
```

Comment:
There are still some vestiges of the version bump in the Info.plist and .xcodeproj files. I think the upgrade script will paper over this inconsistency, but if you can revert these files that’d be great.

Reply:
Should be fixed now